Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp

```diff
@@ -291,121 +322,175 @@ ControlState evdevDevice::Button::GetState() const
 }
 
 evdevDevice::Axis::Axis(u8 index, u16 code, bool upper, libevdev* dev)
```

Comment:
`upper` isn't used anymore, is it?

Reply:
It is still used in the constructor logic.